buildsystem/python.cmake

```diff
@@ -137,7 +138,8 @@ function(add_cython_modules)
 				set_property(GLOBAL APPEND PROPERTY SFT_CYTHON_MODULES_EMBED "${source}")
 				add_executable("${TARGETNAME}" "${CPPNAME}")
 
-				target_link_libraries("${TARGETNAME}" "${PYEXT_PYLIB}")
+				# TODO: use full ldflags and cflags provided by python${VERSION}-config
+				target_link_libraries("${TARGETNAME}" "-L${PYEXT_LIBRARY_DIR} -l${PYEXT_LIBRARY_NAME}")
```

Comment:
I agree, thanks for all of your feedback. To be consistent, I'll use `*_LIBRARY` variable name. It's also unclear why `target_link_libraries` should even take flags...given the name.